Java/Countable/test/CountableTest.java

```diff
@@ -0,0 +1,16 @@
+import org.junit.Test;
+import org.testng.Assert;
+
+public class CountableTest {
+    @Test
+    public void CountTest() throws Throwable {
+        Countable obj1 = new Countable();
+        Countable obj2 = new Countable();
+        Countable obj3 = new Countable();
+
+        Assert.assertEquals(Countable.getTotalCount(), 3);
+
+        obj2.destroy();
+        Assert.assertEquals(Countable.getTotalCount(), 2);
+    }
```

Comment:
You should think about all edge cases. For example, for this class edge case is when all `Countable` objects are destroyed and after that you will try to destroy it again. What should happen in this case?